lib/App/DuckPAN.pm

```diff
@@ -268,21 +270,38 @@ sub execute {
 	my @arr_args = grep { $_ !~ /^-/} @{$args}; # Command line switches make it here, so we try to remove
 	App::DuckPAN::Cmd::Help->run(1) if scalar @arr_args == 0;
 	if (@arr_args) {
-		my @modules;
-		my @left_args;
+		my (@modules, @left_args, $ddg);
 		for (@arr_args) {
-			if ($_ =~ /^www/i ||
-				$_ =~ /^dist/i ||
-				$_ =~ /^ddg/i ||
-				$_ =~ /^app/i) {
+			if (/^www/i ||
+				/^dist/i ||
+				/^(ddg)$/i ||
```

Comment:
Whoops we need to also accept `^ddg::` so that we handle `duckpan DDG::Publisher`, right now that commands falls through to the `emit_and_exit` below.